eml/package.json

```diff
@@ -50,7 +50,7 @@
     "expo-cli": "^6.3.10",
     "expo-file-system": "~14.1.0",
     "expo-font": "~10.2.0",
-    "expo-linear-gradient": "^12.5.0",
+    "expo-linear-gradient": "~11.4.0",
```

Comment:
again here.

Reply:
Didnt downgrade on purpose, i fixed it now